.github/workflows/collect-stats.yaml

```diff
@@ -32,3 +32,11 @@ jobs:
           name: points-dump
           path: data/points-dump
           retention-days: 7
+  keepalive-job:
+    name: Keepalive Workflow
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gautamkrishnar/keepalive-workflow@v2
```

Comment:
I don't think you need these many actions. 

Github disables all workflows because there's no activity in the repo for 60 days.  As per [their README](https://github.com/gautamkrishnar/keepalive-workflow?tab=readme-ov-file#what), this action creates a commit if there's no commit for 50 days. 

So just create a separate workflow file for this that runs once a day at a fixed time.

Reply:
@KShivendu I separate the workflow !!